src/sandbox.rkt

```diff
@@ -377,7 +374,19 @@
                   [start-est start-train-score] [start start-test-score]
                   [target target-cost-score]
                   [result-est end-est-score] [result end-score]
-                  [output (car end-exprs)] [cost-accuracy cost&accuracy])]
+                  [output (car end-exprs)] [cost-accuracy cost&accuracy]))
+
+(define (get-table-data result link)
+  (match-define (job-result test status time _ _ backend) result)
+  (match status
+    ['success
+     (match backend
+      [(improve-result _ _ start targets end _) 
+       (build-table-data-success test start targets end backend result link)]
+      [else ;; HACK Can fail if incomplete reports are saved in *completed-jobs*
+       (dummy-table-row 
+        (job-result test 'incomplete (*timeout*) (timeline-extract) (warning-log) #f)
+        "incomplete" link)])]
```

Comment:
Oof, I'm not thrilled with this logic going in `sandbox.rkt` but it's good to know a future PR fixes this.

Reply:
Are these incomplete jobs filtered out anywhere?

Reply:
This is cleaned up in the next PR.